install

```diff
@@ -21,18 +29,22 @@ main() {
   #local exec_filename=oil.ovm-dbg
   local exec_filename=oil.ovm  # Should this go in configure output?
 
-  local dest="$PREFIX/bin/$exec_filename"
+  local bindir="$PREFIX/bin"
+
+  echo "Installing executables in $DESTDIR/$bindir/"
+  if ! install -v -d "$DESTDIR/$bindir/"; then
+    die "Couldn't create $DESTDIR/$bindir/"
```

Comment:
Why not ust make $DESTDIR part of $bindir, rather than repeat it 4 times?

For expediency I will merge this and fix that ... if I break anything let me know :)